components/builder-api/Cargo.toml

```diff
@@ -35,7 +35,7 @@ serde_json = "*"
 staticfile = "*"
 toml = { version = "*", default-features = false }
 typemap = "*"
-unicase = "*"
+unicase = "1"
```

Comment:
Is this intentional?

Reply:
Yeah there is a compatibility issue that is requiring a "1" for now... will fix/update over the course of the upcoming refactors